integration/integration_test.go

```diff
@@ -4197,11 +4218,17 @@ func testWindowChange(t *testing.T, suite *integrationTestSuite) {
 
 		for i := 0; i < 10; i++ {
 			err = cl.Join(context.TODO(), apidefaults.Namespace, session.ID(sessionID), personB)
+			switch trace.Unwrap(err).(type) {
+			case nil, *ssh.ExitError, *ssh.ExitMissingError:
+				err = nil
+			default:
+				require.NoError(t, err)
+			}
+
 			if err == nil {
 				break
 			}
 		}
```

Comment:
For example, if the `isSSHError` function (see above) was available, this could be rewritten as
```suggestion
			if err == nil || isSSHError(err) {
				err = nil
				break
			}
		}
		require.NoError(t, err)
```

Reply:
Done.